### ISSUE TITLE
pterygoid muscles classification

### ISSUE THREAD

**Issue by markengelstad** [obophenotype/uberon#1138]
![image](https://cloud.githubusercontent.com/assets/6233351/9261095/8e0150a0-41c0-11e5-8456-b56f1974afdc.png)

currently (in hp-edit.owl)
medial pterygoid muscle UBERON:0006718  is subclass of
lateral pterygoid muscle  UBERON:0006719

should be subclass of 
pterygoid muscle  UBERON:0006720

however, when i view these classes in the ontobee hierarchy, they seem correctly classified? (both medial and lateral pterygoid mm are subclasses of pterygoid m.)

**Comment by cmungall** [comment]
OntoBee is showing an ancient uberon, I will file a bug. The core issue was the MPM was asserted to attach to the condylar process - this caused the incorrect classification. OntoBee. I fixed this to just attach to the condyle, back how it was previously. I'm not sure why I temporarily had it on the condylar process. Maybe I made some mistake aligning with FEED? @RDruzinsky did the feed attachments for MPM change in the last 4 months?
